src/Symfony/Component/Validator/Constraints/File.php

```diff
@@ -58,29 +57,57 @@ class File extends Constraint
     public $uploadExtensionErrorMessage = 'A PHP extension caused the upload to fail.';
     public $uploadErrorMessage = 'The file could not be uploaded.';
 
+    protected $maxSize;
```

Comment:
That's the only way to make `__set()` calls work for the property. Isn't it BC since we also provide read access to the property through `__get()`? Otherwise, we would have to move the evaluating of the binary format to the constraint validator.